CHANGELOG.rst

```diff
@@ -2,6 +2,41 @@
 CHANGELOG
 =========
 
+2.7.0
+=====
+
+**ENHANCEMENTS**
+
+* FSx Lustre: Exit with error when failing to retrieve FSx mountpoint
+* `sqswatcher`: The daemon is now compatible with VPC Endpoints so that SQS messages can be passed without traversing
+  the public internet.
+
+**CHANGES**
+
+* Upgrade Nice DCV to version 2020.0-8428.
+* Upgrade Intel MPI to version U7.
+* Upgrade NVIDIA driver to Tesla version 440.64.00.
+* Upgrade EFA installer to version 1.8.4:
+
+  * Kernel module: ``efa-1.5.1`` (no change)
+  * RDMA core: ``rdma-core-25.0`` (no change)
+  * Libfabric: ``libfabric-aws-1.9.0amzn1.1`` (no change)
+  * Open MPI: openmpi40-aws-4.0.3 (updated from openmpi40-aws-4.0.2)
+
+* Download and install Chef client from ParallelCluster's S3 buckets rather than from Chef website when building an AMI.
+
+* Configuration -
+
+  * scheduler parameter: This parameter no longer has a default value and is now mandatory. A deprecation warning
```

Comment:
``base_os`` and ``scheduler`` parameters are now mandatory and they have no longer a default value.
Please remove the part about the warning message.